### PR TITLE
added release notes, changed copyright year

### PR DESCRIPTION
**Pull request by shikhin-metron**
Please review and suggest changes. Let me know if I am missing anything.

# Following is the checklist for PR: 

Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [X] Testing of all the changes has been performed (for bug fixes / features)
- [] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [X] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: release_notes/release_notes.html [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <b>Digital Shadows Release Notes - Published by Digital Shadows Ltd February 05, 2021</b>
```
Comment (devang-metron) [inline release_notes/release_notes.html]: change the date here.

**Review by devang-metron — CHANGES_REQUESTED** [review]
Check if we need to add the 3rd point in release note is required or not

**Comment by shikhin-metron** [comment]
> Check if we need to add the 3rd point in release note is required or not

Yes, I think I will remove it.

File: digital_shadows_connector.py [hunk 2]
```diff
@@ -22,61 +22,149 @@ class DigitalShadowsConnector(BaseConnector):
...
         super(DigitalShadowsConnector, self).__init__()
 
+    def test_connectivity(self):
+        self.save_progress("Testing connectivity")
+        self.save_progress("test")
+        test_connectivity_connector = DSTestConnectivityConnector(self)
+        return test_connectivity_connector.test_connectivity()
+
+    def get_incident_by_id(self, param):
+        self.save_progress("Entered the function")
```
Comment (devang-metron) [inline digital_shadows_connector.py:32]: For all the save_progress function you can give the function name or something related to the function

**Review by devang-metron — CHANGES_REQUESTED** [review]

**Pull request closed without merging**